src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax;
+
+public class Circle extends Figure implements FigureArea {
+    private int radius;
+
+    public Circle(int radius) {

```

Comment:
you should pass color, using constructors in each figure
<img width="264" alt="Screenshot 2022-04-28 at 07 08 14" src="https://user-images.githubusercontent.com/60258373/165674701-6c7bbd00-7680-432d-aad9-5519277ef8ac.png">